### PR TITLE
updated team directory

### PR DESCRIPTION
**Pull request by rbkh**
@maxatbrs pr for additions / deletions / changes to the team on About. thx!

### CODE REVIEW

File: app/_data/team.yml [hunk 1]
```diff
@@ -4,44 +4,15 @@
...
-- first_name: "Chris"
-  last_name: "Bracco"
-  occupation: "Front-End Dev / UI Design"
-  image: "chris-bracco.jpg"
-  pet_image: "chris-bracco.jpg"
-  pet_name: "Gus & Norman"
-  pet_type: "Ferrets"
-  social:
-    - network: "Twitter"
-      url: "https://twitter.com/cbracco"
```
Comment (devinivy) [inline app/_data/team.yml]: :pray: RIP :pray:

File: app/_data/team.yml [hunk 2]
```diff
@@ -135,33 +106,9 @@
       url: "https://github.com/maxatbrs"
       username: "maxatbrs"
 
-- first_name: "Michael"
```
Comment (devinivy) [inline app/_data/team.yml]: :pray: :hibiscus: :pray:

Comment (thebeline) [inline app/_data/team.yml]: Dead to the world...

File: app/_data/team.yml [hunk 3]
```diff
@@ -4,44 +4,15 @@
...
 - first_name: "Brian"
   last_name: "Leighton"
-  occupation: "Web Applications / Robotics"
+  occupation: "Development Manager"
   image: "brian-leighton.jpg"
   pet_image: "brian-leighton.jpg"
   pet_name: "Saxton"
   pet_type: "Saxophone"
 
-- first_name: "Brian"
```
Comment (thebeline) [inline app/_data/team.yml]: Brian is gone too?

**Pull request merged**